pallets/automation-price/src/weights.rs

```diff
@@ -0,0 +1,93 @@
+
+//! Autogenerated weights for `pallet_automation_time`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-05-26, STEPS: `1`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 128
+
+// Executed Command:
+// ./target/release/neumann-collator
+// benchmark pallet
+// --chain
+// dev
+// --execution
+// wasm
+// --wasm-execution
+// compiled
+// --pallet
+// pallet_automation_time
+// --extrinsic
+// "*"
+// --repeat
+// 64
+// --steps
+// 100
+// --output
+// ./pallets/automation-time/src/weights.rs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_automation_time.
+pub trait WeightInfo {
+	fn emit_event() -> Weight;
+	fn reset_asset(v: u32, ) -> Weight;
+	fn update_asset_reset() -> Weight;
+	fn delete_asset_tasks() -> Weight;
+	
+	fn delete_asset_extrinsic() -> Weight;
+	fn asset_update_extrinsic() -> Weight;
+	fn add_asset_extrinsic() -> Weight;
+	fn schedule_swap_task_extrinsic() -> Weight;
+}
+
+/// Weight functions for `pallet_automation_time`.
+pub struct AutomationWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for AutomationWeight<T> {
+	fn emit_event() -> Weight {
+		(20_000_000 as Weight)
+	}
+	fn reset_asset(v: u32, ) -> Weight {
+		(200_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+			.saturating_add((9_000_000 as Weight).saturating_mul(v as Weight))
+			.saturating_add(T::DbWeight::get().reads((3 as Weight).saturating_mul(v as Weight)))
+			.saturating_add(T::DbWeight::get().writes((6 as Weight).saturating_mul(v as Weight)))
+	}
+	fn update_asset_reset() -> Weight{
+		(200_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(1 as Weight))
+			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	}
+	fn delete_asset_tasks() -> Weight{
+		(200_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(1 as Weight))
+			.saturating_add(T::DbWeight::get().writes(3 as Weight))
+	}
+	fn delete_asset_extrinsic() -> Weight{
+		(220_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+			.saturating_add(T::DbWeight::get().writes(6 as Weight))
+	}
+	fn asset_update_extrinsic() -> Weight{
+		(220_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(3 as Weight))
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+			.saturating_add(T::DbWeight::get().reads(21 as Weight))
```

Comment:
this is a ridiculous number because of https://github.com/OAK-Foundation/OAK-blockchain/pull/225/files#diff-1040044567b08cfbb340dcaa8af68f80a19a1ab98db9c94831417c4d982d6eccR658. since the amount that the price increases dictates how much work the chain will do, we have to do the maximum amount. that's why i'm also hesitant to give more than 20 buckets here, so we're currently capped at 20% increase/decreases in asset value